<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration - [Preview]</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Lines and Columns</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nombre de lignes :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="nbLignes">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nombre de colonnes :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="nbColonnes">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Bombs</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>361</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="rbNbBombes">
            <property name="text">
             <string>Nombre de bombes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="nbBombes">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="rbPctBombes">
            <property name="text">
             <string>Pct de bombes (%) </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="pctBombes">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>95</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nbLignes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>setNbLignes(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nbColonnes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>setNbColonnes(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nbBombes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>setNbBombes(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pctBombes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>setPctBombes(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNbBombes</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>nbBombsMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbPctBombes</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>pctBombsMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setNbLignes(int)</slot>
  <slot>setNbColonnes(int)</slot>
  <slot>setNbBombes(int)</slot>
  <slot>setPctBombes(int)</slot>
  <slot>nbBombsMode(bool)</slot>
  <slot>pctBombsMode(bool)</slot>
 </slots>
</ui>
